emloop/cli/eval.py

```diff
@@ -33,9 +34,17 @@ def evaluate(model_path: str, stream_name: str, config_path: Optional[str], cl_a
         validate_config(config)
 
         logging.debug('\tLoaded config: %s', config)
-        
-        main_loop = create_main_loop(config=config, output_root=output_root, restore_from=model_path)
-        main_loop.run_evaluation(stream_name)
 
-    except Exception as ex:  # pylint: disable=broad-except
-        fallback('Evaluation failed', ex)
+        emloop_training = create_emloop_training(
+            config=config, output_root=output_root, restore_from=model_path)
+        if delete_dir:
+            print_delete_warning()
```

Comment:
this should be more or less the first thing that is printed out (i.e. it should be printed out before all the logs from creating the `emloop_training`